src/pages/ClientPage/index.jsx

```diff
@@ -13,15 +16,15 @@ const { Panel } = Collapse;
 
 const { TabPane } = Tabs;
 
-
-
 function ClientPage(props) {
+  const navigate = useNavigate();
+
   const [client, setClient] = React.useState({}); //this is for when we do the fetch from backend
+  const id = useParams().id;
   React.useEffect(() => {
     const token = localStorage.getItem("token");
-
     axios
-      .get("http://localhost:4000/api/clinic/client/1", {
+      .get(`http://localhost:4000/api/clinic/client/${id}`, {
```

Comment:
we should have changed this before but, instead of writing localhost:4000 use the `REACT_APP_API_URL` env